scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/util/SchedulerStarter.java

```diff
@@ -269,14 +270,20 @@ private static boolean isPamrProtocolUsed() {
     }
 
     private static SchedulerAuthenticationInterface startScheduler(CommandLine commandLine, String rmUrl)
-            throws Exception {
+            throws URISyntaxException, InternalSchedulerException, ParseException, SocketException,
+            UnknownHostException, IllegalArgumentException {
         String policyFullName = getPolicyFullName(commandLine);
         logger.info("Starting the scheduler...");
-        SchedulerAuthenticationInterface sai = SchedulerFactory.startLocal(new URI(rmUrl), policyFullName);
-
-        startDiscovery(commandLine, rmUrl);
+        SchedulerAuthenticationInterface sai = null;
+        try {
+            sai = SchedulerFactory.startLocal(new URI(rmUrl), policyFullName);
+            startDiscovery(commandLine, rmUrl);
+            logger.info("The scheduler created on " + sai.getHostURL());
+        } catch (Exception e) {
+            // TODO Auto-generated catch block
+            e.printStackTrace();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use a logger to log this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1148)